docs/git-commands.md

```diff
@@ -0,0 +1,94 @@
+# Git Hub Command Guide
+The following is number of scenarios that you will encounter while contributing to this repo, a convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo, this clone or copy will have the remote name of `origin`.
+
+## Add your own Remote
+Once you have the origin remote you will need to add a remote to your own fork of the repo. This can be done with the following command, just replace the placeholder with your own github name.
+
+It should also be noted you will need to have forked the repo, how to do this can be seen in Tutorial.md
```

Comment:
I think you're going to add the link to tutorial once the other pr is merged? It is merged now btw. :)